src/oc-id/config/settings.yml

```diff
@@ -1,5 +1,5 @@
 chef:
-  endpoint: https://192.168.33.100
+  endpoint: https://33.33.33.10
```

Comment:
is this a local-environment configuration change or is it meant to be checked in?